src/Microsoft.AspNetCore/project.json

```diff
@@ -0,0 +1,27 @@
+﻿{
+  "version": "1.0.1-*",
+
+  "dependencies": {
+    "NETStandard.Library": "1.6.0",
+
+    "Microsoft.AspNetCore.Diagnostics": "1.0.0",
+    "Microsoft.AspNetCore.Hosting": "1.0.0",
+    "Microsoft.AspNetCore.StaticFiles": "1.0.0",
+
+    "Microsoft.Extensions.Configuration.EnvironmentVariables": "1.0.0",
+    "Microsoft.Extensions.Configuration.Json": "1.0.0",
+    "Microsoft.Extensions.Logging": "1.0.0",
+    "Microsoft.Extensions.Logging.Console": "1.0.0",
+    "Microsoft.Extensions.Options.ConfigurationExtensions": "1.0.0",
+    "Microsoft.Extensions.Configuration.FileExtensions": "1.0.0",
+
+    "Microsoft.AspNetCore.Routing": "1.0.1",
```

Comment:
🙁  Why is Routing different from StaticFiles? Seems nicer to just alphabetize them like we do with every other repo.